app/src/main/java/grinnell/appdev/edu/lyles/fragments/SnacksFragment.java

```diff
@@ -3,28 +3,103 @@
 
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import org.json.JSONArray;
+
+import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
+
+import grinnell.appdev.edu.lyles.AsyncRetrieval;
+import grinnell.appdev.edu.lyles.ItemAdapter;
+import grinnell.appdev.edu.lyles.Constants;
+import grinnell.appdev.edu.lyles.MenuItem;
 import grinnell.appdev.edu.lyles.R;
 
+import static grinnell.appdev.edu.lyles.R.layout.fragment_snacks;
+
 /**
  * A fragment to display available hot food to user
  */
 public class SnacksFragment extends Fragment {
```

Comment:
This class could be generalized, along with the Hot Food, Drinks, and possibly Beer fragments. See if you can adjust it so the constructor takes the unique components (ie. the JSON url). You could also use an enum and then a switch case if that's easier.